client/src/components/ui.js

```diff
@@ -313,6 +313,27 @@ function TextArea({ styles, ...props }) {
   return <textarea {...props} css={{ ...styleInput, ...styles }} />;
 }
 
+const styleScroll = {
```

Comment:
Thanks for moving this style for the scroll to the `ui.js` file

Reply:
You're welcome `:)`